stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -749,8 +757,9 @@ function completions(string::String, pos::Int, context_module::Module=Main)
 
         if inc_tag === :string &&
            length(paths) == 1 &&  # Only close if there's a single choice,
-           !isdir(expanduser(replace(string[startpos:prevind(string, first(r))] * paths[1].path,
-                                     r"\\ " => " "))) &&  # except if it's a directory
+           try !isdir(expanduser(replace(string[startpos:prevind(string, first(r))] * paths[1].path,
```

Comment:
Maybe this should be pulled out in a function (perhaps local to the function). It's starting to become hard to read.

Reply:
Sure, the accumulation of things is a bit too much indeed

Reply:
Moved to the new `close_path_completion` function